app/models/user.rb

```diff
@@ -3,6 +3,7 @@ class User < ApplicationRecord
   validates :email, uniqueness: true, presence: true
   has_secure_password
   has_many :orders
+  belongs_to :merchant, optional: true 
```

Comment:
@johnktravers what does this optional parameter do?

Reply:
Optional makes the foreign key optional. Because not every user will be connected to a merchant (only the merchant employees and merchant admins will), we want the merchant_id column to be NULL for default users and admins. This allows that to happen

Reply:
Ah, got it. Thanks!

Reply:
Good to know!